src/containers/Tasks/Tasks.js

```diff
@@ -35,46 +35,44 @@ import {
 } from '@carbon/icons-react';
 
 import { ListPageLayout } from '..';
-import { fetchTasks } from '../../actions/tasks';
+import { fetchTasks as fetchTasksActionCreator } from '../../actions/tasks';
 import { deleteTask } from '../../api';
 import {
   getSelectedNamespace,
   getTasks,
   getTasksErrorMessage,
   isFetchingTasks,
-  isReadOnly,
-  isWebSocketConnected
+  isWebSocketConnected,
+  isReadOnly as selectIsReadOnly
 } from '../../reducers';
 
-export /* istanbul ignore next */ class Tasks extends Component {
-  state = {
-    deleteError: null,
-    showDeleteModal: false,
-    toBeDeleted: []
-  };
+/* istanbul ignore next */
+function Tasks(props) {
+  const {
+    filters,
+    namespace,
+    fetchTasks,
+    error,
+    intl,
+    tasks,
+    loading,
+    isReadOnly,
+    namespace: selectedNamespace,
+    webSocketConnected
+  } = props;
 
-  componentDidMount() {
-    document.title = getTitle({ page: 'Tasks' });
-    this.fetchData();
-  }
+  const [deleteError, setDeleteError] = useState(null);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [toBeDeleted, setToBeDeleted] = useState([]);
+  const [cancelSelection, setCancelSelection] = useState(() => {
+    return () => {};
+  });
```

Comment:
the initial value can be `null` here

Reply:
@AlanGreene I've explained below. :)

Reply:
It should still be safe to set the initial value to null here, this would match the previous behavior of the class-based component where there was no initial value for this state value.